packages/react-native-oursky/src/NetworkFailureToast.js

```diff
@@ -0,0 +1,134 @@
+// @flow
+import * as React from "react";
+import { SafeAreaView, StyleSheet, View, Image, NetInfo } from "react-native";
+import Text from "./Text";
+import { TextStyle, ViewStyle } from "./styles";
+import FadeAnimation from "./FadeAnimation";
+
+const styles = StyleSheet.create({
+  safeArea: {
+    bottom: 0,
+    left: 0,
+    right: 0,
+    position: "absolute",
+    alignItems: "center",
+  },
+  toastContainer: {
+    backgroundColor: "white",
+    width: 349,
```

Comment:
Will it be too large for some devices. For example, iPhone SE which has width 320pt